### PR TITLE
[IMP] ecommerce: updated catalog page

### PR DESCRIPTION
**Pull request by toaa-odoo**
previous version was based on V12, forward to 15.

taskid: 3061068

### CODE REVIEW

File: content/applications/websites/ecommerce/getting_started/catalog.rst [hunk 1]
```diff
@@ -1,37 +1,99 @@
...
 
-All your published items show up in your catalog page (or *Shop* page).
+Categorize the product catalog
+==============================
 
-.. image:: catalog/shop.png
+In Odoo, there is a **specific category model** for your eCommerce. This category, which can be
+found on the product form, in the :guilabel:`eCommerce` tab, is a many-to-many field. This allows a
+single product to be added in multiple eCommerce categories, and therefore show up in multiple
+categories on your website.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/getting_started/catalog.rst:16]:
```suggestion
In Odoo, there is a **specific category model** for your eCommerce. This category, which can be
found on the product form in the :guilabel:`eCommerce` tab, is a many-to-many field. This allows a
single product to be added to multiple eCommerce categories and appear in multiple ones on your website.
```

File: content/applications/websites/ecommerce/getting_started/catalog.rst [hunk 2]
```diff
@@ -1,37 +1,99 @@
...
+.. note::
+   The **eCommerce category** must *not* be mistaken with the :guilabel:`Product Category` found in
+   the :guilabel:`General Information` tab, which serves another purpose.
 
-Most options are available in the *Customize* menu: display attributes,
-website categories, etc.
+When your product is configured, go to your **Website** page, click on :guilabel:`Customize` and
+enable the :guilabel:`eCommerce Categories`. This option creates a categories search menu for your
+customers to use. Additionally, you can also enable :guilabel:`Show Sort By` to allow sorting on the
+products page by :guilabel:`Catalog price` or :guilabel:`Name`.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/getting_started/catalog.rst:29]:
```suggestion
When your product is configured, go to your **Website** page, click on :guilabel:`Customize` and
enable the :guilabel:`eCommerce Categories`. This option creates a categories search menu for your
customers to use. Additionally, you can enable :guilabel:`Show Sort By` to allow sorting on the
products page by :guilabel:`Catalog price` or :guilabel:`Name`.
```

File: content/applications/websites/ecommerce/getting_started/catalog.rst [hunk 3]
```diff
@@ -1,37 +1,99 @@
...
+color of a product, the material, etc.).
+
+To do so, click on the :guilabel:`Customize` button in the top-left corner of your **website** page.
 
-Highlight a product
-===================
+.. tip::
+   You can use **attribute filters** even if you do not work with product variants. When adding
+   attribute to your products, make sure to only specify *one* value per attribute. Odoo will not
+   create variants if no combination is possible.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/getting_started/catalog.rst:47]:
```suggestion
.. tip::
   You can use **attribute filters** even if you do not work with product variants. When adding
   attributes to your products, make sure only to specify *one* value per attribute. Odoo will not
   create variants if no combination is possible.
```

File: content/applications/websites/ecommerce/getting_started/catalog.rst [hunk 4]
```diff
@@ -1,37 +1,99 @@
...
+
+.. image:: catalog/ecommerce-highlight.png
+   :align: center
+   :alt: Product highlight options.
+
+Ribbons
+-------
+
+It is possible to add a ribbon to the product's view-form to indicate the product is on sale, new,
+etc. Simply follow the steps of the :ref:`Highlights paragraph <getting_started/catalog/highlights>`
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/getting_started/catalog.rst:67]:
```suggestion
You can highlight products to make them more visible on the **main** product page. Make the product
image bigger, push it to the top of the page, add a ribbon, etc. To do so, from the **main page** of the
shop, click :guilabel:`Edit`, and select the product you wish to modify. Click :guilabel:`Style` in
the right panel to modify the product's view form.

.. image:: catalog/ecommerce-highlight.png
   :align: center
   :alt: Product highlight options.

Ribbons
-------

It is possible to add a ribbon to the product's view form to indicate the product is on sale, new,
etc. Simply follow the steps of the :ref:`Highlights paragraph <getting_started/catalog/highlights>`
```

File: content/applications/websites/ecommerce/getting_started/catalog.rst [hunk 5]
```diff
@@ -1,37 +1,99 @@
...
+
+Design the category page
+========================
+
+You can customize the layout of the category page using the website builder. Note that editing the
+layout of the category page is global, editing one category layout will affect all category pages.
+
+.. image:: catalog/ecommerce-layout.png
+   :align: center
+   :alt: Layout options of the category pages.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/getting_started/catalog.rst:82]:
```suggestion
and select the **ribbon** type you wish to use.

.. image:: catalog/ecommerce-ribbons.png
   :align: center
   :alt: The different product ribbons.

Design the category page
========================

You can customize the layout of the category page using the website builder. Note that editing the
layout of the category page is global; editing one category layout will affect all category pages.

.. image:: catalog/ecommerce-layout.png
   :align: center
   :alt: Layout options of the category pages.
```

File: content/applications/websites/ecommerce/getting_started/catalog.rst [hunk 6]
```diff
@@ -1,37 +1,99 @@
...
-.. image:: catalog/shop_list.png
-   :align: center
\ No newline at end of file
+- :guilabel:`Add to cart`: adds a button to **add to cart** the product when hoovering the mouse
+  over the product's view-form;
+- :guilabel:`Comparison List`: adds a button to **compare** products based on their price, variant,
+  etc;
+- :guilabel:`Product Description`: shows the **product's description** when hoovering over the
+  product's view form;
+- :guilabel:`Wishlist Button`: adds a button to **wishlist** the product.
```
Comment (jaae-odoo) [inline content/applications/websites/ecommerce/getting_started/catalog.rst:99]:
```suggestion
- :guilabel:`Add to cart`: adds a button to **add to cart** the product when hoovering the mouse
  over the product's view form;
- :guilabel:`Comparison List`: adds a button to **compare** products based on their price, variant,
  etc;
- :guilabel:`Product Description`: shows the **product's description** when hoovering over the
  product's view form;
- :guilabel:`Wishlist Button`: adds a button to **wishlist** the product.
```

**Review by jaae-odoo — CHANGES_REQUESTED** [review]

File: content/applications/websites/ecommerce/getting_started/catalog.rst (hunk 1 above)
Comment (toaa-odoo) [reply content/applications/websites/ecommerce/getting_started/catalog.rst:16]: Changed it up a bit, replaced your "ones" suggestion w/ "categories" to make sure there's no misunderstanding.

**Review by jaae-odoo — APPROVED** [review]

**Pull request closed without merging**